backend/endpoints/messages.py

```diff
@@ -1,4 +1,5 @@
 from fastapi import APIRouter, BackgroundTasks, HTTPException, status
+from fastapi_pagination import Page, add_pagination, paginate
```

Comment:
@bolexs how are you using paginate? What's the behaviour and did you test this implementation with data from the DB? I am asking because I can see a caveat in the [documentation](https://pypi.org/project/fastapi-pagination/), that warns against using the default paginate method when working with databases.